src/agent/LibFuzzerDotnetLoader/Program.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Reflection;
+using System.Runtime;
+using System.Runtime.InteropServices;
+using System.Text;
+
+using TestOneSpan = SharpFuzz.ReadOnlySpanAction;
+delegate void TestOneArray(byte[] data);
+
+namespace LibFuzzerDotnetLoader {
+    public class Program {
+        public static void Main(string[] args) {
+            var target = Environment.GetEnvironmentVariable("LIBFUZZER_DOTNET_TARGET");
+
+            if (target == null) {
+                throw new Exception("`LIBFUZZER_DOTNET_TARGET` not set. " +
+                                    "Expected format: \"<assembly-path>:<class>:<static-method>\"");
+            }
+
+            var parts = target.Split(':');
+
+            var assemPath = parts[0];
```

Comment:
We should probably fail helpfully and with context if any of these accesses are out of bounds, but we can revisit that.

Reply:
Tracked in #2130.